### PR TITLE
Added makefile

### PR DESCRIPTION
**Pull request by ma3a**

### CODE REVIEW

**Review by Witherking25 — CHANGES_REQUESTED** [review]
I think it needs some edits, grammatically, and path wise

File: makefile [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+DECK_PASS := $(shell node -p "require('./.vscode/settings.json').deckpass")
+DECK_SSH_KEY := $(shell node -p "require('./.vscode/settings.json').deckkey")
+DECK_USER_HOMEDIR := $(shell node -p "require('./.vscode/settings.json').deckdir")
+
+
+all: pnpm_setup prepaire_dist
+pnpm_setup:
+	pnpm i	
+
+prepaire_dist:
```
[the remark this answers is not in the document]
Comment (ma3a) [reply makefile]: Yeah! Thanks - fixed!

File: makefile [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+	rm -rf ./distribution
+	mkdir ./distribution
+	mkdir ./distribution/dist
```
[the remark this answers is not in the document]
Comment (ma3a) [reply makefile]: Added `$(BUILD_DIR)` variable for this

**Review by Witherking25 — APPROVED** [review]
This seems ready to merge

**Pull request merged**